documentation/modules/configuring/con-config-mirrormaker2-high-volume-messages.adoc

```diff
@@ -6,25 +6,24 @@
 = Handling high volumes of messages
 
 [role="_abstract"]
-If your MirrorMaker 2.0 deployment is going to be handling a high volume of messages, you might need to adjust its configuration to support it. The configuration can help manage throughput and issues with latency. 
+If your MirrorMaker 2.0 deployment is going to be handling a high volume of messages, you might need to adjust its configuration to support it. 
+The configuration can help manage throughput and issues with latency. 
 
-MirrorMaker 2.0 uses an internal producer client (`KafkaProducer`) to replicate data from the source to target Kafka cluster. 
+MirrorMaker 2.0 uses an internal producer client to replicate data from the source to the target Kafka cluster. 
 To control the streaming of high volumes of data, you can tune the following producer configuration for optimal throughput: 
 
 * Producer batch size 
-* Producer buffer size 
+* Producer buffer memory 
```

Comment:
I would remove all buffer memory mentions (lines 16, 26, 47).

Decreasing this value doesn't directly address the issue (slow producer) and it is also complicated, as you would need to take into account the avg message size.